### PR TITLE
MIRA

### PR DESCRIPTION
**Pull request by mvandeberg**

### CODE REVIEW

**Comment by jredbeard** [comment]
🎉

File: libraries/chain/database.cpp [hunk 1]
```diff
@@ -3088,6 +3122,12 @@ void database::_apply_block( const signed_block& next_block )
    clear_expired_transactions();
    clear_expired_orders();
    clear_expired_delegations();
+
+   if( next_block.block_num() % 100000 == 0 )
```
Comment (sgerbino) [inline libraries/chain/database.cpp]: Useless conditional

File: libraries/chain/include/steem/chain/history_object.hpp [hunk 2]
```diff
@@ -67,6 +72,8 @@ namespace steem { namespace chain {
             c( *this );
          }
 
+         account_history_object() {}
```
Comment (sgerbino) [inline libraries/chain/include/steem/chain/history_object.hpp]: Should we standardize using STEEM_STD_ALLOCATOR_CONSTRUCTOR?

File: libraries/plugins/account_by_key/include/steem/plugins/account_by_key/account_by_key_objects.hpp [hunk 3]
```diff
@@ -26,6 +24,8 @@ class key_lookup_object : public object< key_lookup_object_type, key_lookup_obje
          c( *this );
       }
 
+      key_lookup_object() {}
```
Comment (sgerbino) [inline libraries/plugins/account_by_key/include/steem/plugins/account_by_key/account_by_key_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR?

File: libraries/plugins/block_log_info/include/steem/plugins/block_log_info/block_log_info_objects.hpp [hunk 4]
```diff
@@ -29,6 +29,8 @@ class block_log_hash_state_object : public object< block_log_hash_state_object_t
          c( *this );
       }
 
+      block_log_hash_state_object() {}
```
Comment (sgerbino) [inline libraries/plugins/block_log_info/include/steem/plugins/block_log_info/block_log_info_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/block_log_info/include/steem/plugins/block_log_info/block_log_info_objects.hpp [hunk 5]
```diff
@@ -53,6 +55,8 @@ class block_log_pending_message_object : public object< block_log_pending_messag
          c( *this );
       }
 
+      block_log_pending_message_object() {}
```
Comment (sgerbino) [inline libraries/plugins/block_log_info/include/steem/plugins/block_log_info/block_log_info_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/follow/include/steem/plugins/follow/follow_objects.hpp [hunk 6]
```diff
@@ -116,6 +116,8 @@ class blog_author_stats_object : public object< blog_author_stats_object_type, b
          c( *this );
       }
 
+      blog_author_stats_object() {}
```
Comment (sgerbino) [inline libraries/plugins/follow/include/steem/plugins/follow/follow_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/market_history/include/steem/plugins/market_history/market_history_plugin.hpp [hunk 7]
```diff
@@ -122,6 +121,8 @@ struct order_history_object : public object< order_history_object_type, order_hi
       c( *this );
    }
 
+   order_history_object() {}
```
Comment (sgerbino) [inline libraries/plugins/market_history/include/steem/plugins/market_history/market_history_plugin.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 8]
```diff
@@ -45,6 +43,8 @@ class rc_resource_param_object : public object< rc_resource_param_object_type, r
          c( *this );
       }
 
+      rc_resource_param_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 9]
```diff
@@ -59,6 +59,8 @@ class rc_pool_object : public object< rc_pool_object_type, rc_pool_object >
          c( *this );
       }
 
+      rc_pool_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 10]
```diff
@@ -73,6 +75,8 @@ class rc_account_object : public object< rc_account_object_type, rc_account_obje
          c( *this );
       }
 
+      rc_account_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 11]
```diff
@@ -96,6 +100,8 @@ class rc_delegation_pool_object : public object< rc_delegation_pool_object_type,
          c( *this );
       }
 
+      rc_delegation_pool_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 12]
```diff
@@ -114,6 +120,8 @@ class rc_indel_edge_object : public object< rc_indel_edge_object_type, rc_indel_
          c( *this );
       }
 
+      rc_indel_edge_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp [hunk 13]
```diff
@@ -144,6 +152,8 @@ class rc_outdel_drc_edge_object : public object< rc_outdel_drc_edge_object_type,
          c( *this );
       }
 
+      rc_outdel_drc_edge_object() {}
```
Comment (sgerbino) [inline libraries/plugins/rc/include/steem/plugins/rc/rc_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

File: libraries/plugins/transaction_status/include/steem/plugins/transaction_status/transaction_status_objects.hpp [hunk 14]
```diff
@@ -31,15 +31,15 @@ enum transaction_status
...
-   transaction_status_object() = delete;
-
 public:
    template< typename Constructor, typename Allocator >
    transaction_status_object( Constructor&& c, allocator< Allocator > a )
    {
       c( *this );
    }
 
+   transaction_status_object() {}
```
Comment (sgerbino) [inline libraries/plugins/transaction_status/include/steem/plugins/transaction_status/transaction_status_objects.hpp]: STEEM_STD_ALLOCATOR_CONSTRUCTOR

**Pull request merged**